source/intro.md

```diff
@@ -760,6 +760,36 @@ more easily understood format.
 Visualizations are a great tool for summarizing information to help you
 effectively communicate with your audience.
 
+## Adding new columns using `assign`
+
```

Comment:
this breaks the flow of the chapter. right before this the text is leading up to creating viz, and then it switches to assign